superset-frontend/src/types/Dashboard.ts

```diff
@@ -19,7 +19,7 @@
 import Owner from './Owner';
 import Role from './Role';
 
-type Dashboard = {
+export interface Dashboard {
```

Comment:
what is this change for? Dashboard doesn't have methods why prefer interface?

Reply:
Interface is always preferred other than in React props.

Reply:
fwiw i think there's still a bunch of discrepancies across Superset for this. I don't really care one way or another (consistency is better in my mind) but noting that even after this change there will still be a bunch of inconsistencies